internal/pipeline/pipeline.go

```diff
@@ -228,6 +248,68 @@ func (f *Function) Validate() error {
 	return nil
 }
 
+// runner returns a fnRunner from the image and configs of
+// this function.
+func (f *Function) runner() (kio.Filter, error) {
+	config, err := f.config()
+	if err != nil {
+		return nil, err
+	}
+	return &fnRunner{
+		fn: &runtime.ContainerFn{
+			Image: f.Image,
+		},
+		fnConfig: config,
+	}, nil
+}
+
+func (f *Function) config() (*yaml.RNode, error) {
+	var node *yaml.RNode
+	switch {
+	case f.ConfigPath != "":
```

Comment:
Is there a TODO to special handle these resources?

    # 'configPath' specifies a relative path to a file in the current directory containing a K8S resource used
    # as the function config.
    # This resource is excluded when resolving 'sources', and as a result
    # cannot be operated on by the pipeline.

Reply:
We haven't implemented the function to resolve the `sources`. I think here is not the good place to add such todo.

Reply:
Can you add a TODO where it's most appropriate? This is a signal to reviewers that this is not yet done instead of a an oversight. It also makes it easy to grep for TODOs to figure out what's left to do.